solutions/swb-reference/integration-tests/tests/isolated/sshKeys/get.test.ts

```diff
@@ -11,29 +12,74 @@ import { checkHttpError } from '../../../support/utils/utilities';
 describe('Get Key Pair negative tests', () => {
   const setup: Setup = new Setup();
   let adminSession: ClientSession;
+  //let adminUserId: string | undefined;
+  let project: { id: string };
 
-  beforeEach(async () => {
+  beforeAll(async () => {
     adminSession = await setup.getDefaultAdminSession();
+    //adminUserId = adminSession.getUserId();
+    const { data: costCenter } = await adminSession.resources.costCenters.create({
+      name: 'test cost center',
+      accountId: setup.getSettings().get('defaultHostingAccountId'),
+      description: 'a test object'
+    });
+
+    const { data } = await adminSession.resources.projects.create({
+      name: `TestProject-${uuidv4()}`,
+      description: 'Project for list users for project tests',
+      costCenterId: costCenter.id
+    });
+
+    project = data;
+  });
+
+  beforeEach(async () => {
     expect.hasAssertions();
   });
 
   afterEach(async () => {
     await setup.cleanup();
   });
 
+  describe('with User that is not authorized', () => {
+    let invalidUserId: string;
+    beforeEach(() => {
+      invalidUserId = '00000000-0000-0000-0000-000000000000';
```

Comment:
We'll have to use the user sessions for this test and not mocking the `userId`, but it is on the right track. This test is blocked on getting the users set up with unique sessions (like we set up `adminSession`). That will be what I hope you work on next!

Reply:
got it. will update in another PR